README.md

```diff
@@ -103,8 +104,11 @@ by adding -DUSE_CPU_ONLY=1 to the cmake command line.
     cd leela-zero
     git submodule update --init --recursive
 
-    # Install build depedencies
-    sudo apt install libboost-dev libboost-program-options-dev libboost-filesystem-dev opencl-headers ocl-icd-libopencl1 ocl-icd-opencl-dev zlib1g-dev
+    # Install leelaz build depedencies
+    sudo apt install cmake libboost-dev libboost-program-options-dev libboost-filesystem-dev opencl-headers ocl-icd-libopencl1 ocl-icd-opencl-dev zlib1g-dev curl
+    
+    # Optional : if you want to use autogtp to contribute, install autogtp dependencies as well
```

Comment:
Reword as: ..if you want to contribute to the distributed effort, you need these dependencies as well...

Reply:
ok